### PR TITLE
bhyve: use lowercase b for bhyve

### PR DESCRIPTION
**Pull request by grahamperrin**
My points of reference for consistent use of lowercase include this year's <https://freebsdfoundation.org/wp-content/uploads/2023/05/a66452da2b483e5e92bde6533b7236c4d11382eb-1.pdf> (22 of 22 lowercase).

### CODE REVIEW

**Comment by grahamperrin** [comment]
Thanks to Corvin Köhne for comments via email (2023-08-21), cc @bsdjhb at the time.

**Comment by ckoehne** [comment]
1. The common commit message style is something like `bhyve: use lowercase b for bhyve`
2. I don't think that we want links to private repos in the commit message
3. Why don't you fix all wrong usages of bhyve I've send you? E.g.: `sys/dev/virtio/pci/virtio_pci_legacy.c: * the specification, but VirtualBox, QEMU/KVM, and BHyVe seem to`

**Comment by grahamperrin** [comment]
@ckoehne thanks, I wasn't sure whether all were wrong. I'll get to this maybe later today.

**Comment by ckoehne** [comment]
https://wiki.freebsd.org/bhyve#Q:_Is_it_.22bhyve.22.2C_.22Bhyve.22.2C_.22BHyVe.22_or_BHyve.3F

**Comment by grahamperrin** [comment]
https://github.com/freebsd/freebsd-src/pull/838.patch for a plain text view of the revised commit message. 

10 files changed, 11 insertions(+), 11 deletions(-)

File: usr.sbin/bhyve/vmgenc.c
Comment (ckoehne) [inline usr.sbin/bhyve/vmgenc.c:93]: If I read the ACPI specs correctly, this is an invalid _HID value. See https://uefi.org/htmlspecs/ACPI_Spec_6_4_html/06_Device_Configuration/Device_Configuration.html#hid-hardware-id

I don't know who is familiar with ACPI and can help out here.

**Review by ckoehne — APPROVED** [review]
Besides the wrong _HID value, which is another issue and shouldn't block this PR, LGTM.

**Comment by bsdimp** [comment]
I don't think this is the correct change. Some of them should be capitalized. And some of the mixed case might be right...
So I'm just going to close this as invalid. Please socialize this in the bhyve mailing list and get one of the committers there to commit whatever the consensus is there.

**Pull request closed without merging**
